static/app/components/sidebar/index.tsx

```diff
@@ -182,6 +182,15 @@ function Sidebar() {
     organization,
   };
 
+  const isFloatingSidebar = horizontal || collapsed;
+
+  const nestedSidebarItemProps = {
+    ...sidebarItemProps,
+    isNested: true,
+    isFloatingSidebar,
+    orientation: 'left' as SidebarOrientation,
```

Comment:
Hm, why is this always `left`?

Reply:
If we let don't hardcode this, when its in the mobile view, orientation would automatically switch to 'top', and the accordion will look like this.
![image](https://github.com/getsentry/sentry/assets/44422760/2503d348-b0dd-42c6-ad15-1ac4942296ca)

Reply:
This should only apply to nested items, because we want the main items to show their icon instead of the full name.